apps/src/dance/Dance.js

```diff
@@ -176,16 +179,34 @@ Dance.prototype.init = function (config) {
 
   ReactDOM.render(
     <Provider store={getStore()}>
-      <AppView
-        visualizationColumn={
-          <DanceVisualizationColumn
-            showFinishButton={showFinishButton}
-            setSong={this.setSongCallback.bind(this)}
-            resetProgram={this.reset.bind(this)}
-          />
-        }
-        onMount={onMount}
-      />
+      <ErrorBoundary
+        // this is actually the Lab2 Error Fallback page. We may want to refactor this after Hour of Code.
+        fallback={<ErrorFallbackPage />}
+        onError={(error, componentStack) => {
+          const state = getStore().getState();
+          Lab2MetricsReporter.updateProperties({
```

Comment:
Can we set this in `init`? That way if we use this reporter elsewhere, these fields will already be set

Reply:
Done!